From 82e5b493add498c3fdbeaa55174639b76353b78b Mon Sep 17 00:00:00 2001
From: Dave Gerlach <d-gerlach@ti.com>
Date: Mon, 28 Jul 2014 19:35:57 -0500
Subject: [PATCH 0931/1587] ARM: dts: am335x-evm: Remove matrix_keypad_default
 from pinmux default

We do not need the pinmux driver to configure the pins for the matrix
keypad for us, the driver can handle this itself.

Signed-off-by: Dave Gerlach <d-gerlach@ti.com>
Signed-off-by: Sekhar Nori <nsekhar@ti.com>
[zou: Original patch taken from
ti-sdk-am335x-evm-08.00.00.00-Linux-x86-Install.bin]
Signed-off-by: Cao Zou <cao.zou@windriver.com>
---
 arch/arm/boot/dts/am335x-evm.dts |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/am335x-evm.dts b/arch/arm/boot/dts/am335x-evm.dts
index 1ced317..c603ccb 100644
--- a/arch/arm/boot/dts/am335x-evm.dts
+++ b/arch/arm/boot/dts/am335x-evm.dts
@@ -167,7 +167,7 @@
 
 &am33xx_pinmux {
 	pinctrl-names = "default";
-	pinctrl-0 = <&matrix_keypad_default &volume_keys_s0 &clkout2_pin>;
+	pinctrl-0 = <&volume_keys_s0 &clkout2_pin>;
 
 	matrix_keypad_default: matrix_keypad_default {
 		pinctrl-single,pins = <
-- 
1.7.5.4

